php-practice_1.php

```diff
@@ -1,41 +1,218 @@
 <?php
 // Q1 変数と文字列
+$name = '秋田';
 
 
+$message = '私の名前は「'.$name.'」です。';
+echo $message;
+
 // Q2 四則演算
+$x = 5;
+$y = 4;
 
+$num = $x*$y;
+echo $num ;
+echo "\n";
+echo $num /2;
 
 // Q3 日付操作
-
+$message='現在時刻は、'.date('Y年m月d日 H時i分s秒').'です。';
+echo $message;
 
 // Q4 条件分岐-1 if文
+$device = '';
+
+if ($device === 'windows' || $device == 'mac') {
+    echo'使用OSは、'.$device.'です。';
+
+} else { 
+    echo 'どちらでもありません。';
+}
 
 
 // Q5 条件分岐-2 三項演算子
+$age = 18;
+$message = ($age >= 18) ? '成人です。' : '未成年です。';
 
+echo $message;
 
 // Q6 配列
+$kanto = [
+    '東京都',
+    '神奈川県',
+    '栃木県',
+    '千葉県',
+    '埼玉県',
+    '茨城県',
+    '群馬県'
+];
+
+echo $kanto[2].'と'.$kanto[3].'は関東地方の都道府県です。';
+
 
 
 // Q7 連想配列-1
+// $array2 = [
+//     '新宿区',
+//     '横浜市',
+//     '千葉市',
+//     'さいたま市',
+//     '宇都宮市',
+//     '前橋市',
+//     '水戸市'
+// ];
+
+
+$array = [
+    // key => value,
+    '東京都' => '新宿区',
+    '神奈川県' => '横浜市',
+    '千葉県' => '千葉市',
+    '埼玉県' => 'さいたま市',
+    '栃木県' => '宇都宮市',
+    '群馬県' => '前橋市',
+    '茨城県' => '水戸市'
+];
+
+// echo $array['東京都'];
+foreach ($array as $prefecture => $capitalCity) {
+    echo $capitalCity . "\n";
+}
 
 
 // Q8 連想配列-2
+$array = [
+    // key => value,
+    '東京都' => '新宿区',
+    '神奈川県' => '横浜市',
+    '千葉県' => '千葉市',
+    '埼玉県' => 'さいたま市',
+    '栃木県' => '宇都宮市',
+    '群馬県' => '前橋市',
+    '茨城県' => '水戸市'
+];
+
+// echo $array['東京都'];
+foreach ($array as $prefecture => $capitalCity) {
+     // もしキーの$prefectureが埼玉県だったら
+     if ($prefecture == '埼玉県') {
+        echo $prefecturen='埼玉県'.'の県庁所在地は、'.$capitalCity='さいたま市' . 'です。';
+    } 
+     
+}
 
 
 // Q9 連想配列-3
+$array = [
+    // key => value,
+    '東京都' => '新宿区',
+    '神奈川県' => '横浜市',
+    '千葉県' => '千葉市',
+    '埼玉県' => 'さいたま市',
+    '栃木県' => '宇都宮市',
+    '群馬県' => '前橋市',
+    '茨城県' => '水戸市',
+    '愛知県' => '名古屋市',
+    '大阪府' =>'大阪市'
+];
+
+
+
+// echo $array['東京都'];
+foreach ($array as $prefecture => $capitalCity) {
+    // var_dump($prefecture);
+     // もしキーの$prefectureが埼玉県だったら
+     if ($prefecture === '東京都' || $prefecture === '神奈川県'|| $prefecture === '千葉県'|| $prefecture === '埼玉県'|| $prefecture === '栃木県'|| $prefecture === '群馬県'|| $prefecture === '茨城県') {
```

Comment:
このようにコードが横に長くなる場合、改行がおすすめです。
理由は、読みやすさと、Gitの管理のしやすさです。
Gitはコードの変更を行で感知するため、1行が長いと、どこかが変更された時にその行の中で何が変わったのかが分かりづらくなってしまいます！
```suggestion
     if (
     　$prefecture === '東京都' ||
     　$prefecture === '神奈川県'||
     　$prefecture === '千葉県'||
     　$prefecture === '埼玉県'||
     　$prefecture === '栃木県'||
     　$prefecture === '群馬県'||
     　$prefecture === '茨城県'
     ) {
```